tensorflow_datasets/core/features/tensor_feature.py

```diff
@@ -131,19 +132,10 @@ def encode_example(self, example_data):
                        f'For {self}')
 
     np_dtype = np.dtype(self.dtype.as_numpy_dtype)
-    if isinstance(example_data, tf.Tensor):
-      raise TypeError(
-          f'Error encoding: {example_data!r}. `_generate_examples` should '
-          'yield `np.array` compatible values, not `tf.Tensor`')
-    if not isinstance(example_data, np.ndarray):
-      example_data = np.array(example_data, dtype=np_dtype)
-    # Ensure the shape and dtype match
-    if example_data.dtype != np_dtype:
```

Comment:
We want to keep this check too

Reply:
Maybe I miss something, but we just converted it to `np.ndarray`. This test condition should always be false. Shouldn't it?

Reply:
See my above comment